<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>310</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>410</width>
     <height>310</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0,0,0">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>70</height>
           </size>
          </property>
          <property name="title">
           <string>Figure type:</string>
          </property>
          <widget class="QRadioButton" name="rBStar">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>84</width>
             <height>19</height>
            </rect>
           </property>
           <property name="text">
            <string>Star</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QRadioButton" name="rBPolygon">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>84</width>
             <height>19</height>
            </rect>
           </property>
           <property name="text">
            <string>Polygon</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>90</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>90</height>
           </size>
          </property>
          <property name="title">
           <string>Gradient colors:</string>
          </property>
          <widget class="QPushButton" name="pBColor1">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>80</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Color 1</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pBColor2">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>60</y>
             <width>80</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Color 2</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Border color:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pBBorder">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Vertex number:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sBVertex">
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>15</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Figures number:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sBFigures">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Angle:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sBAngle">
            <property name="maximum">
             <number>360</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Outer radius/inner radius ratio:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="ratio">
            <property name="minimum">
             <double>1.200000000000000</double>
            </property>
            <property name="maximum">
             <double>6.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.400000000000000</double>
            </property>
            <property name="value">
             <double>1.600000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Filled area:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="hSFilled">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>67</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Stars:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
